src/EnergyPlus/DataZoneEquipment.cc

```diff
@@ -1810,12 +1811,14 @@ void ZoneEquipmentMixer::setOutletConditions(EnergyPlusData &state)
         sumEnthalpy += spaceOutletNode.Enthalpy * mixerSpace.fraction;
         sumHumRat += spaceOutletNode.HumRat * mixerSpace.fraction;
         sumCO2 += spaceOutletNode.CO2 * mixerSpace.fraction;
+        sumGenContam += spaceOutletNode.GenContam * mixerSpace.fraction;
```

Comment:
Most (all?) other places that work with CO2 and GenContam have ifs like `if (state.dataContaminantBalance->Contaminant.GenericContamSimulation) {`. Should there be ifs here?

Reply:
@mjwitte Ok, I've added the IF checks as per your suggestion and just now made a commit (as well as merged in develop to get everything back up to speed).  Anticipating that this should come back all green as before.